lib/graphql-filter.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
-const Filter = require('broccoli-filter');
+const Filter = require('broccoli-persistent-filter');
 const gql = require('graphql-tag');
 
+gql.disableFragmentWarnings();
```

Comment:
just curious, what was this part about?

Reply:
Ah, I totally forgot I'd made that change. I hadn't actually meant to include it in this PR, and I'm happy to pull it into a separate one if you'd like.

Apollo likes fragment names to be unique, and they pushed a warning about that into their parsing layer (it used to be at runtime, and you'd get a warning [if you used the same fragment twice](https://github.com/apollographql/apollo-client/issues/940) in different queries at all).

Because the lifetime of `gql-tag`'s module state spans multiple builds, if you make changes to a fragment, the rebuild issues a warning because it's now parsed multiple different versions of that fragment.